plan/plan_loader.go

```diff
@@ -551,3 +555,23 @@ func ValidatePlans(plans []blip.Plan) error {
 
 	return nil
 }
+
+func deepcopyPlan(p *blip.Plan) (blip.Plan, error) {
+	// Since the operation needed here is not performance critical we can piggy back off of gob
+	// encode/decode to make the deep copy and not rely on staying up-to-date with the blip.Plan's
+	// internal structure.
```

Comment:
Also good comment: this isn't perf-critical and avoiding the hassle of struct changes is a good choice for this kind of approach.